### PR TITLE
Add Confd v0.16.0 (new formula)

### PR DESCRIPTION
**Pull request by qinxin0720**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/master/CONTRIBUTING.md)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?

-----

### CODE REVIEW

File: Formula/Confd.rb [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+    desc "Manage local application configuration files using templates and data from etcd or consul"
+    homepage "https://github.com/kelseyhightower/confd"
+    url "https://github.com/kelseyhightower/confd/archive/v0.16.0.tar.gz"
+    sha256 "4a6c4d87fab77aa9827370541024a365aa6b4c8c25a3a9cab52f95ba6b9a97ea"
+    head "https://github.com/kelseyhightower/confd.git"
+  
+    depends_on "go" => :build
+  
+    def install
+      ENV["GOPATH"] = buildpath
```
Comment (SMillerDev) [inline Formula/Confd.rb]: https://github.com/Homebrew/homebrew-core/issues/47627

Please try to use modules instead of GOPATH

Comment (qinxin0720) [reply Formula/Confd.rb]: I am sorry, this project does not use the go modules to manage dependencies

**Comment by Bo98** [comment]
Can you make the filename all lowercase? `confd.rb`

**Comment by qinxin0720** [comment]
> Can you make the filename all lowercase? `confd.rb`

sure

**Comment by Bo98** [comment]
Thanks @qinxin0720

**Pull request closed without merging**